charts/istio-gateway/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for common.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+gatewayClass: "istio-ingressgateway"
+replicaCount: 2
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using Release.Name
+  name: ""
+
+resources:
+  limits:
+    cpu: 2000m
```

Comment:
We decided long ago not to put CPU limits. We can discuss and revisit at some point. Does this really need to be here?